client/.eslintrc

```diff
@@ -2,10 +2,10 @@
   "parser": "babel-eslint",
   "extends": "airbnb",
   "settings": {
-    import/resolver: "webpack"
+    "import/resolver": "webpack"
```

Comment:
Oh I didn't remove all of the settings ><  Sorry this is my bad, should be more clear.